src/lib/y2partitioner/dialogs/partition_size.rb

```diff
@@ -1,7 +1,43 @@
+require "y2storage"
 require "yast"
 require "cwm/dialog"
 require "ui/greasemonkey"
 
+Yast.import "Popup"
+
+module Y2Storage
+  # Monkey-patched Region to get a size
+  class Region
+    def size
+      block_size * length
+    end
+
+    def last
+      start + length - 1
+    end
+
+    # @return [Boolean] is *block* inside the region?
+    def cover?(block)
```

Comment:
document what is block, as I have no clue what it is and it looks for me more like specific byte at disk.